src/Sulu/Bundle/MediaBundle/EventListener/CacheInvalidationListener.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\EventListener;
+
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Sulu\Bundle\MediaBundle\Entity\File;
+use Sulu\Bundle\MediaBundle\Entity\FileVersion;
+use Sulu\Bundle\MediaBundle\Entity\FileVersionMeta;
+use Sulu\Bundle\MediaBundle\Entity\MediaInterface;
+use Sulu\Component\HttpCache\HandlerInvalidateReferenceInterface;
+
+/**
+ * Invalidate references when account/contact are persisted.
+ */
+class CacheInvalidationListener
```

Comment:
Can't we have one of these listeners, instead of adding one in each bundle?

Reply:
in which bundle would you add that?

Reply:
HttpCacheBundle?